nautobot/docs/additional-features/jobs.md

```diff
@@ -304,6 +304,10 @@ Messages recorded with `log()` or `log_debug()` will appear in a job's results b
 
 Markdown rendering is supported for log messages.
 
+!!! note
+    Using `self.log_failure()` marks a specific log as a failure, but it will **not** stop the execution of the job. In order to fail the job utilize a Python `raise` or `return` value.
+
+
```

Comment:
This doesn't flow as cleanly as I'd like relative to the above several paragraphs (particularly the other mention of failure logging at line 301). Perhaps restructure them to something like the following:

> Messages recorded with `log()` or `log_debug()` will appear in a job's results but are never associated with a particular object; the other `log_*` functions may be invoked with or without a provided object to associate the message with.
>
> It is advised to log a message for each object that is evaluated so that the results will reflect how many objects are being manipulated or reported on.
>
> Markdown rendering is supported for log messages.
>
> !!! note
>    Using `self.log_failure()`, in addition to recording a log message, will flag the overall job as failed, but it will **not** automatically stop the execution of the job. To end a job early, you can use a Python `raise` or `return` as appropriate.

Reply:
@jeffkala following up, is this something you can still take care of?

Reply:
@lampwins yes, sorry for the delay.  This one slipped my mind.

Reply:
@lampwins @glennmatthews restructured as your recommendation showed.  Let me know if anything else is needed.